react/tic-tac-toe/src/app/screens/Game/utils.js

```diff
@@ -0,0 +1,20 @@
+export function calculateWinner(squares) {
+  const lines = [[0, 1, 2], [3, 4, 5], [6, 7, 8], [0, 3, 6], [1, 4, 7], [2, 5, 8], [0, 4, 8], [2, 4, 6]];
+  for (let i = 0; i < lines.length; i++) {
+    const [a, b, c] = lines[i];
+    if (squares[a] && squares[a] === squares[b] && squares[a] === squares[c]) {
+      return squares[a];
+    }
+  }
+  return null;
+}
+
+export function getWinner(winner, xIsNext) {
+  let status = null;
```

Comment:
You could define this function as an arrow function and use a ternary condition to return the first text or the other one